rosco-web/src/main/groovy/com/netflix/spinnaker/rosco/controllers/BakeryController.groovy

```diff
@@ -50,7 +53,27 @@ class BakeryController {
   @Value('${defaultCloudProviderType:aws}')
   BakeRequest.CloudProviderType defaultCloudProviderType
 
-  @RequestMapping(value = '/{region}/bake', method = RequestMethod.POST)
+  @RequestMapping(value = '/bakeOptions', method = RequestMethod.GET)
+  List<BakeOptions> bakeOptions() {
+    cloudProviderBakeHandlerRegistry.list().collect { it.getBakeOptions() }
+  }
+
+  @RequestMapping(value = '/bakeOptions/{cloudProvider}', method = RequestMethod.GET)
+  BakeOptions bakeOptionsByCloudProvider(@PathVariable("cloudProvider") BakeRequest.CloudProviderType cloudProvider) {
+    def bakeHandler = cloudProviderBakeHandlerRegistry.lookup(cloudProvider)
+    if (!bakeHandler) {
```

Comment:
@ttomsu The param is an enum, so the only way this can be true is if the enum is updated to support a new cloud provider, but the cloud provider does not register itself.